docs/support/user-account-migration.md

```diff
@@ -1,10 +1,10 @@
-# Common issues for GCC 1.0 and SHIP-HATS users
+# Common issues for GCC 1.0 MDM onboarding and SHIP-HATS account migration users
 
 1. I received an email stating, "Your account was used for a new registration request". What does it mean?
 
-You may receive this email if you are a GCC 1.0 user or part of the SHIP-HATS account migration, who tried to sign up for TechPass account when you already have an active account.
+You may receive this email if you are a GCC 1.0 user or a part of the SHIP-HATS account migration, who tried to sign up for TechPass account when you already have an active account.
 
-If you did not re-attempt to sign up for TechPass account, this could be a potential hack and hence [report this incident][service-request] to us and monitor your TechPass account for suspicious activities.
+If you neither re-attempted to sign up for TechPass account nor part of the SHIP-HATS account migration , this could be a potential hack and hence [report this incident][service-request] to us and monitor your TechPass account for suspicious activities.
```

Comment:
I'm trying to remove the statement this is a potential hack. It's a possibility but may not be. so we just need get the user to monitor their activities without explicitly stating it is.

Reply:
yeah you are right. Done!